tutorial/20_recipes/006_user_defined_pruner.py

```diff
@@ -16,24 +16,25 @@
 The :func:`~optuna.study.create_study` constructor takes, as an optional
 argument, a pruner inheriting from :class:`~optuna.pruners.BasePruner`. The
 pruner should implement the abstract method
-:meth:`~optuna.pruners.BaseSampler.prune`, which takes arguments for the
+:func:`~optuna.pruners.BaseSampler.prune`, which takes arguments for the
```

Comment:
> :func:`~optuna.pruners.BaseSampler.prune`

Should :func:`~optuna.pruners.BaseSampler.prune` be replaced with `:func:`~optuna.pruners.BasePruner.prune`? Sorry if I misunderstand something.

Reply:
You are right, good catch.

Reply:
Thank @himkt for your careful check and @crcrpar for your double-check!